Doc/tutorial/classes.rst

```diff
@@ -4,6 +4,12 @@
 Classes
 *******
 
+Classes provide a means of bundling data and functionality together.  Creating
```

Comment:
No, I think starting with a brief intro of what classes are, like this paragraph here, is good.